interview-questions.md

```diff
@@ -9,49 +9,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 Your answer:
+A conditional statement is a 
 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
 Your answer:
+git is a system that supports version control.
 
 Researched answer:
+Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.
 
 2. Which JavaScript operators will return a Boolean value?
 
 Your answer:
+Comparitve  and Logical operators.
 
 Researched answer:
+Boolean expressions use relational and logical operators. The result of a Boolean expression is either true or false. Boolean expressions allow us to write programs that decide whether to execute some code or not.
 
 3. What is an index? What is the difference between index and value?
 
 Your answer:
+An index is the position of a value, a value is any character.
 
 Researched answer:
+The index property is a read-only property. It contains the position of a regular expression match within a string.n a JavaScript environment, those chunks are called values. Though all values are made of bits, they play different roles. Every value has a type that determines its role. Some values are numbers, some values are pieces of text, some values are functions, and so on. To create a value, you must merely invoke its name.
 
```

Comment:
Researched answers can be supported by resources, but need to be in your own words. We want to know what you understand and how you can describe an index and value, not someone else's explanation. 

https://www.w3resource.com/javascript/object-property-method/array-index.php
https://eloquentjavascript.net/01_values.html